keyform/urls.py

```diff
@@ -5,6 +5,9 @@
 
 urlpatterns = [
     url(r'^$', views.HomeView.as_view(), name='home'),
+    url(r'^contact', views.ContactView.as_view(), name='contact'),
+    url(r'^edit-contact/(?P<pk>\d+)$', views.EditContactView.as_view(), name='edit_contact'),
+    url(r'^create-contact', views.NewContactView.as_view(), name='create_contact'),
```

Comment:
`edit_contact` and `create_contact` should be changed to `edit-contact` and `create-contact` respectively.